### DISCUSSION TITLE
Use PostgreSQL for Development

### DISCUSSION THREAD

**Discussion by moritzlerch** [pretix/pretix#3437] [Q&A] [answered]
Hey, 

I'm currently doing plugin development for pretix. For this I would like to use the configuration of a ticketshop that is already running in production on my server. I'm running pretix within docker and I'm using PostgreSQL as my database.

For development purposes I would like to use the database on my server in my local development. For this I've created a backup of the db with `pg_dump`. After some research I could not find a way to get the existing database running in the development environment. Is there any way to accomplish this?

Many thanks in advance!

**Comment by raphaelm** [comment, marked as answer]
I'm not sure I understand the question, the configuration works the same in development and production, just pop a pretix.cfg in your working directory and fill it.
